packages/documentation/src/stories/components/collapsible/collapsible.stories.ts

```diff
@@ -29,7 +30,7 @@ export default meta;
 // DECORATORS
 function externalControls(story: StoryFn, context: StoryContext) {
   const { args, canvasElement } = context;
-  const togglerId = `${context.id}--button`;
+  const togglerId = `button--${context.id}`;
```

Comment:
This is not valid otherwise, as we are targetting an id starting with a number later.